modules/bastion/scripts/oracle.template.sh

```diff
@@ -3,172 +3,8 @@
 # Copyright 2017, 2019, Oracle Corporation and/or affiliates.  All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl
 
-# Hardening of the host based on https://www.cisecurity.org/benchmark/oracle_linux/
-# This script is used only if the Oracle Linux platform image is used to create the bastion
-
 yum update --security
 
 sed -i -e "s/autoinstall\s=\sno/# autoinstall = yes/g" /etc/uptrack/uptrack.conf
```

Comment:
This `sed` command changes the `autoinstall = no` line (i.e., don't auto-install) in `/etc/uptrack/uptrack.conf` to `# autoinstall = yes`. After the change, the line is commented out; so the effect is still the same as "don't auto-install", right? What am I missing?

Reply:
Not sure why either. Probably something that slipped during testing of a module or another. Will remove the comment. Thanks for applying your eagle eyes!

Reply:
fixed